### ISSUE TITLE
"microsoft/dotnet:2.2-aspnetcore-runtime" image has 2.2.0-preview3-27014-02 version of dotnet instead of 2.2

### ISSUE THREAD

**Issue by FrameBassman** [dotnet/dotnet-docker#854]
## Steps to reproduce the issue
1. Create dockerfile:
```
FROM microsoft/dotnet:2.2-aspnetcore-runtime
RUN dotnet --info
```
2. Run docker build from this dockerfile:
```
docker build .
```

## Expected behavior
dotnet version should be equals to "2.2"

## Actual behavior
dotnet version is "2.2.0-preview3-27014-02"

## Output of `docker version`

```
Client:
 Version:           18.06.0-ce
 API version:       1.30 (downgraded from 1.38)
 Go version:        go1.10.3
 Git commit:        0ffa825
 Built:             unknown-buildtime
 OS/Arch:           darwin/amd64
 Experimental:      false

Server:
 Engine:
  Version:          17.06.2-ce
  API version:      1.30 (minimum version 1.12)
  Go version:       go1.8.3
  Git commit:       cec0b72
  Built:            Tue Sep  5 19:59:19 2017
  OS/Arch:          linux/amd64
  Experimental:     false
```

## Output of `docker info`

```
Containers: 1
 Running: 0
 Paused: 0
 Stopped: 1
Images: 243
Server Version: 17.06.2-ce
Storage Driver: aufs
 Root Dir: /mnt/sda1/var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 316
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 6e23458c129b551d5c9871e5174f6b1b7f6d1170
runc version: 810190ceaa507aa2727d7ae6f4790c76ec150bd2
init version: 949e6fa
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.4.86-boot2docker
Operating System: Boot2Docker 17.06.2-ce (TCL 7.2); HEAD : ff16afa - Wed Sep  6 00:17:25 UTC 2017
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 1.955GiB
Name: dinghy
ID: 52YB:BC4W:QD5Y:ZXUU:IYZS:GW5V:JILS:A7LR:ACJ3:OJY7:P55E:O5KT
Docker Root Dir: /mnt/sda1/var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 16
 Goroutines: 27
 System Time: 2018-12-04T15:28:31.864522387Z
 EventsListeners: 0
Username: framebassman
Registry: https://index.docker.io/v1/
Labels:
 provider=xhyve
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Comment by FrameBassman** [comment]
Solved: please, use "microsoft/dotnet:2.2.0-aspnetcore-runtime" image

**Comment by MichaelSimons** [comment]
The `microsoft/dotnet:2.2-aspnetcore-runtime` is supported and contains 2.2.0.  I suspect you have an older version of the tag cached.  Try building your Dockerfile with the `--pull` option or execute the following

```
$ docker pull microsoft/dotnet:2.2-aspnetcore-runtime
2.2-aspnetcore-runtime: Pulling from microsoft/dotnet
a5a6f2f73cd8: Pull complete
35ceebdba949: Pull complete
04c3e576ee8a: Pull complete
3e26d475595a: Pull complete
Digest: sha256:c532d3c6521c4880fdefe8502f986f88deef48c0c5cfbdfc9c412b7aee8ca07d
Status: Downloaded newer image for microsoft/dotnet:2.2-aspnetcore-runtime

$ docker run -it --rm microsoft/dotnet:2.2-aspnetcore-
runtime dotnet --info

Host (useful for support):
  Version: 2.2.0
  Commit:  1249f08fed

.NET Core SDKs installed:
  No SDKs were found.

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.2.0 [/usr/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.2.0 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.2.0 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```
